src/main/utils/resource.ts

```diff
@@ -82,7 +82,7 @@ export class Resource {
         let tmpFolder: string = ScanUtils.createTmpDir();
         try {
```

Comment:
`ScanUtils.removeFolder(tmpFolder);` needs await?

Reply:
yes, `fse.remove` is async method this is why there was a bug before